lib/plox/date_time_scale.ex

```diff
@@ -12,10 +12,18 @@ defmodule Plox.DateTimeScale do
   @type datetime :: DateTime.t() | NaiveDateTime.t()
 
   @doc """
-  Creates a new `Plox.DateTimeScale` struct
+  Creates a new `Plox.DateTimeScale` struct.
 
   Accepts 2 elixir `DateTime` or `NaiveDateTime` structs as `first` and `last`.
   Negative ranges are not currently supported.
+
```

Comment:
Would be good to note that an error is raised when given values for arguments that don't satisfy the `DateTime`/`NaiveDateTime` struct requirement:

```suggestion
  Creates a new `Plox.DateTimeScale` struct.

  Accepts `first` and `last` datetime structs (`t:DateTime.t/0` or `t:NaiveDateTime.t/0`) as the basis for the scale's bounds.

  An error is raised, if either `first` or `last` is not a `DateTime` or `NaiveDateTime`.

  Negative ranges are not currently supported.

```